lib/lita/handlers/artifactory.rb

```diff
@@ -126,6 +126,11 @@ def promote(response)
           EOH
         end
 
+        # This event allows other lita utilities to act when projects are promoted.
+        robot.trigger(:artifactory_artifact_promoted, product: project, version: version)
```

Comment:
Would we really want to raise an error when sending an event? We could just specify "unknown" or something...